lib/LDA.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+import numpy
+from lib.content_based import ContentBased
+from sklearn.decomposition import LatentDirichletAllocation
+from sklearn.feature_extraction.text import CountVectorizer
+
+
+class LDARecommender(ContentBased):
```

Comment:
Still I didn't merge your AbstractRecommender, I opened issue #8 for it, and will do it later.